kiosk/public/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <!-- <link rel="icon" href="kiosk-data/favicon.ico" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="MakeCode Arcade Kiosk mode. Showcase games made on Microsoft MakeCode Arcade in a carousel list."
+    />
+    <!-- <link rel="apple-touch-icon" href="kiosk-data/logo192.png" /> -->
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="kiosk-data/manifest.json" />
```

Comment:
I'm missing what this kiosk-data path is; is the intention to swap this out as part of build similar to static?

Reply:
Ah now I see https://github.com/microsoft/pxt-arcade/pull/6009/files, I believe manifest.json should be considered 'part of the build' and versioned as such ideally (rather than to docs), but not enough to block this on / can do as follow up